database/utils/logging.js

```diff
@@ -0,0 +1,51 @@
+const ProgressBar = require('progress');
+const { createLogger, format, transports } = require('winston');
+const { combine, timestamp, label, printf } = format;
+
+const winstonLogFormat = printf(({ message, label, timestamp }) => {
+  return `${timestamp} [${label}]: ${message}`;
+});
+
+const winstonLogger = createLogger({
+  level: 'info',
+  format: combine(label({ label: 'INFO' }), timestamp(), winstonLogFormat),
+  transports: [
+    // - Write all logs with level `error` and below to `error.log`
+    new transports.File({
+      filename: 'error.log',
+      level: 'error',
+      format: combine(label({ label: 'ERROR' }), timestamp(), winstonLogFormat)
+    }),
+
+    // - Write all logs with level `info` and below to `combined.log`
+    new transports.File({ filename: 'combined.log' })
+  ]
+});
+
+const generateProgressBar = (progressBarLength) => {
+  const bar = new ProgressBar(
+    '  Migrating :current/:total records at :rate records/s | :percent complete | ETA :etas | [:bar]',
+    {
+      complete: '█',
```

Comment:
What is this character? Should that be a space?

Reply:
Thats just a fancy bar, that displays a progress kind of like when you do a `npm install` See: https://www.google.com/search?q=unicode+progress+bar+characters